README.md

```diff
@@ -138,6 +147,25 @@ archive.append('hello.txt', function () {
 })
 ```
 
+#### `archive.checkpoint(version, [callback])`
+
+Write a version checkpoint to the archive, marking the state for future reference. Version includes:
+
+```js
+{
+  label: '1.0.0', // can be any string, but semantic versions are recommended
+  message: 'The first version!' // an optional string describing the checkpoint
+}
+```
+
+If the label has been used before, Hyperdrive will error.
```

Comment:
I think `version` might be fine in this API call, but I think the message schema should still use `label`. This is the schema:

```
message Checkpoint {
  required string label = 1;
  required uint64 timestamp = 2;
  optional string message = 3;
}
```

`version` wouldn't make sense. It should be `id` or `name`.